diffs/preTestAaveV2_Ethereum_DAIParamsUpdates_20230817_postTestAaveV2_Ethereum_DAIParamsUpdates_20230817.md

```diff
@@ -0,0 +1,52 @@
+## Reserve changes
+
+### Reserves altered
+
+#### DAI ([0x6B175474E89094C44Da98b954EedeAC495271d0F](https://etherscan.io/address/0x6B175474E89094C44Da98b954EedeAC495271d0F))
+
+| description | value before | value after |
+| --- | --- | --- |
+| reserveFactor | 15 % | 25 % |
+| stableBorrowRateEnabled | true | false |
+| interestRateStrategy | [0xfffE32106A68aA3eD39CcCE673B646423EEaB62a](https://etherscan.io/address/0xfffE32106A68aA3eD39CcCE673B646423EEaB62a) | [0xc6A068E321C83FEacc25C80118E2B1208c54B6ce](https://etherscan.io/address/0xc6A068E321C83FEacc25C80118E2B1208c54B6ce) |
+| optimalUsageRatio | 80 % | 90 % |
+| maxExcessUsageRatio | 20 % | 10 % |
+| variableRateSlope1 | 4 % | 5 % |
+| interestRate | ![before](/.assets/c415a8c57745a62d63e11134fe8acf5059377542.svg) | ![after](/.assets/d1a493171bfb16a17ecaf7a2d7ef8e0782317be5.svg) |
```

Comment:
for some reason looks like these assets don't exist. Maybe forgot to push them? (Maybe need to re run tests to generate again and push new assets?)

Reply:
assets working correctly locally can't reproduce issue.